Generator/Generator.php

```diff
@@ -72,7 +72,8 @@ protected function renderFile($template, $target, $parameters)
      */
     public static function mkdir($dir, $mode = 0777, $recursive = true)
     {
-        if (@mkdir($dir, $mode, $recursive)) {
+        if (!is_dir(dirname($dir))) {
+            mkdir($dir, $mode, $recursive);
```

Comment:
Please keep the silencing operator (Symfony always does `@mkdir`) because eventually an error will happen here.